packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -224,7 +241,12 @@ export function publish(options: InitOptions): void {
 
 
   uiEvents.forEach((eventConfig => {
-    const { eventName, cacheElementReactInfo = false } = eventConfig;
+    const { eventName } = eventConfig;
+    if (eventName === 'mouseover' && eventConfig.durationThresholdToEmitHoverEvent != null) {
+      // Publish hover events as well,  if configured
+      ALHoverPublisher.publish({ channel, uiEvents });
+    }
```

Comment:
This causing circular dependency between these two modules. Also, later if we wanted to break things down into plugins, it will be harder to manager. 

It is better to just call the `ALHoverPublisher.publish({ channel, uiEvents });` in the AutoLogging.ts directly. (https://github.com/facebook/hyperion/blob/main/packages/hyperion-autologging/src/AutoLogging.ts#L156-L170). 

That way each one of these modules are both independent, but also linked to `uiEventPublisher` option.